packages/ui/react-ui-editor/src/extensions/markdown/formatting.ts

```diff
@@ -1124,7 +1131,17 @@ export const getFormatting = (state: EditorState): Formatting => {
   };
 
   const { selection } = state;
+  const commentState = state.field(commentsState);
+
   for (const range of selection.ranges) {
+    if (!comment) {
+      const overlap = (a: Range, b: SelectionRange): boolean => {
+        return a.from <= b.to && a.to >= b.from;
+      };
```

Comment:
I wonder if we have one of these helpers already. I don't want to be re-creating this function every selection change. @richburdon thoughts on where this should live?